From 343182516e0e09e5c07e92ea24b002b299277700 Mon Sep 17 00:00:00 2001
From: Andrew Cooper <andrew.cooper3@citrix.com>
Date: Mon, 28 Jul 2025 12:44:25 +0100
Subject: tools/flask: Use tabs uniformly

Most indentation is with tabs, but a few spaces have slipped in.  Switch them
back to tabs.

No functional change.

Signed-off-by: Andrew Cooper <andrew.cooper3@citrix.com>
Acked-by: Daniel P. Smith <dpsmith@apertussolutions.com>

diff --git a/tools/flask/policy/modules/xen.if b/tools/flask/policy/modules/xen.if
index def60da88301..42df92939923 100644
--- a/tools/flask/policy/modules/xen.if
+++ b/tools/flask/policy/modules/xen.if
@@ -94,7 +94,7 @@ define(`manage_domain', `
 			getaddrsize pause unpause trigger shutdown destroy
 			setaffinity setdomainmaxmem getscheduler resume
 			setpodtarget getpodtarget getpagingmempool setpagingmempool };
-    allow $1 $2:domain2 { set_vnumainfo dt_overlay };
+	allow $1 $2:domain2 { set_vnumainfo dt_overlay };
 ')
 
 # migrate_domain_out(priv, target)
@@ -181,9 +181,9 @@ define(`make_device_model', `
 #   Allow a device to be used by a domain
 #   only if an IOMMU provides isolation.
 define(`use_device_iommu', `
-    allow $1 $1_self:mmu exchange;
-    allow $1 $2:resource use_iommu;
-    allow $1 domio_t:mmu { map_read map_write };
+	allow $1 $1_self:mmu exchange;
+	allow $1 $2:resource use_iommu;
+	allow $1 domio_t:mmu { map_read map_write };
 ')
 
 # use_device_iommu_nointremap(domain, device)
@@ -192,30 +192,30 @@ define(`use_device_iommu', `
 #   interrupt remapping.
 #   Allows acceptance of (typically older) less isolating hardware.
 define(`use_device_iommu_nointremap', `
-    allow $1 $1_self:mmu exchange;
-    allow $1 $2:resource { use_iommu use_iommu_nointremap };
-    allow $1 domio_t:mmu { map_read map_write };
+	allow $1 $1_self:mmu exchange;
+	allow $1 $2:resource { use_iommu use_iommu_nointremap };
+	allow $1 domio_t:mmu { map_read map_write };
 ')
 
 # use_device_noiommu(domain, device)
 #   Allow a device to be used by a domain
 #   even without an IOMMU available.
 define(`use_device_noiommu', `
-    allow $1 $1_self:mmu exchange;
-    allow $1 $2:resource { use_iommu use_iommu_nointremap use_noiommu };
-    allow $1 domio_t:mmu { map_read map_write };
+	allow $1 $1_self:mmu exchange;
+	allow $1 $2:resource { use_iommu use_iommu_nointremap use_noiommu };
+	allow $1 domio_t:mmu { map_read map_write };
 ')
 
 # admin_device(domain, device)
 #   Allow a device to be used and delegated by a domain
 define(`admin_device', `
-    allow $1 $2:resource { setup stat_device add_device add_irq add_iomem add_ioport remove_device remove_irq remove_iomem remove_ioport plug unplug };
-    allow $1 $2:hvm bind_irq;
-    use_device_noiommu($1, $2)
+	allow $1 $2:resource { setup stat_device add_device add_irq add_iomem add_ioport remove_device remove_irq remove_iomem remove_ioport plug unplug };
+	allow $1 $2:hvm bind_irq;
+	use_device_noiommu($1, $2)
 ')
 
 # delegate_devices(priv-domain, target-domain)
 #   Allow devices to be delegated
 define(`delegate_devices', `
-    allow $1 $2:resource { add remove };
+	allow $1 $2:resource { add remove };
 ')
